src/rules/declaration-block-no-redundant-longhand-properties/index.js

```diff
@@ -0,0 +1,85 @@
+import {
+  isEqual,
+  isString,
+  transform,
+} from "lodash"
+import {
+  optionsMatches,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+import shorthandData from "../../reference/shorthandData"
+
+export const ruleName = "declaration-block-no-redundant-longhand-properties"
+
+export const messages = ruleMessages(ruleName, {
+  rejected: (props) => (
+    `Unexpected longhand properties "${props}"`
```

Comment:
Whatever you both agree on is fine by me. I just thought it's worth a suggestion. The `Expected` version is shorter and clearer — though perhaps less consistent.